opm/simulators/flow/BlackoilModelParametersEbos.hpp

```diff
@@ -31,6 +32,10 @@ namespace TTag {
 struct FlowModelParameters {};
 }
 
+// forward declaration to make this header usable from eclbasevanguard.hh
+template<class TypeTag, class MyTypeTag>
+struct EclDeckFileName;
+
 template<class TypeTag, class MyTypeTag>
```

Comment:
Where is this used in eclbasevanguard.hh?

Reply:
Anoher remnant of the typetag fight. At the time I thought I could instantiate BlakoilModelParametersEbos in EclEclipseGrid. That or the thought that I had skills to do it turned out to be a quite naive assumption